app/views/layouts/shared/_navbar.html.erb

```diff
@@ -1,20 +1,20 @@
 <nav>
   <%= link_to root_path do %>
-    <%= image_tag "artify_logo", :height => "40", class: "logo"%>
+    <%= image_tag "artify_logo", class:"logo"%>
   <% end %>
   <a href='#' data-item='Rent a Piece'>Rent a Piece</a>
   <%# <a href='#' data-item='Collection'>Collection</a>
   <a href='#' data-item='Top Rented'>Top Rented</a>
   <a href='#' data-item='Contact-us'>Contact-us</a> %>
-  <%= render partial: "pages/shared/searchbar", class: "search-form"%>
+
```

Comment:
Where is the form template ?
You don't need to have a functionnal form but we still need a form to be present for visualization. Even if it does nothing